src/login/register.jsx

```diff
@@ -13,6 +13,7 @@ const Register = () => {
   const [password, setPassword] = useState("");
   const [confirmPassword, setconfirmPassword] = useState("");
   const [isRegistering, setIsRegistering] = useState(false);
+  const [age, setAge] = useState("");
```

Comment:
Are you using double quotes or single quotes everywhere? 
Could you check the linter corrects/flags when using both?

Reply:
Yes I will double check that